pkg/mime/mime.go

```diff
@@ -36,9 +37,13 @@ func init() {
 
 // RegisterMime is a package level function that registers
 // a mime type with the given extension.
-// TODO(labkode): check that we do not override mime type mappings?
```

Comment:
The original `TODO` suggests that it may or may not be a problem to override a type mapping. As such, a change here would be justified only by some evidence that there is a real problem to be fixed. This is not the case as far as I know.

Furthermore, any caller of this function would need to be amended to handle the error now returned.